generators/client/templates/gulpfile.js

```diff
@@ -279,11 +294,14 @@ gulp.task('watch', function () {
     gulp.watch(<% if(useSass) { %>config.sassSrc<% } else { %>config.app + 'content/css/**/*.css'<% } %>, ['styles']);
     gulp.watch(config.app + 'content/images/**', ['images']);
     gulp.watch(config.app + 'app/**/*.js', ['inject']);
+    <%_ if(useSass) { _%>
+    gulp.watch(config.app + 'app/**/*.{scss,sass}', ['inject:sass']);
+    <%_ } _%>
     gulp.watch([config.app + '*.html', config.app + 'app/**', config.app + 'i18n/**']).on('change', browserSync.reload);
 });
 
 gulp.task('install', function () {
-    runSequence(['wiredep', 'ngconstant:dev'<% if(useSass) { %>, 'sass'<% } %><% if(enableTranslation) { %>, 'languages'<% } %>], 'inject');
+    runSequence(['wiredep', 'ngconstant:dev'<% if(useSass) { %>, 'inject:sass', 'sass'<% } %><% if(enableTranslation) { %>, 'languages'<% } %>], 'inject');
```

Comment:
Inject:sass should be before sass task, i mean should be before the square brace start as things inside square brace are run in parellel